src/library/librarycontrol.cpp

```diff
@@ -212,6 +212,8 @@ LibraryControl::LibraryControl(Library* pLibrary)
     m_pSortOrder = std::make_unique<ControlPushButton>(ConfigKey("[Library]", "sort_order"));
     m_pSortOrder->setButtonMode(ControlPushButton::TOGGLE);
     m_pSortColumnToggle = std::make_unique<ControlEncoder>(ConfigKey("[Library]", "sort_column_toggle"), false);
+    m_pSortFocusedColumn = std::make_unique<ControlPushButton>(
+            ConfigKey("[Library]", "sort_focused_column"));
```

Comment:
It would match better to the internal nomenclature 
```suggestion
            ConfigKey("[Library]", "sort_current_column"));
```
But I don't really mind. I just want to make sure the name is well considered.

Reply:
OK, makes sense .